.github/workflows/autolabeler.yml

```
@@ -0,0 +1,31 @@
+  cancel-in-progress: true
+
+jobs:
+  autolabel:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: release-drafter/release-drafter@v7
+        with:
+          disable-releaser: true
```

actionlint (1.7.12)

[error] input "disable-releaser" is not defined in action "release-drafter/release-drafter@v7". available inputs are "commitish", "config-name", "dry-run", "filter-by-range", "footer", "header", "include-pre-releases", "initial-commits-since", "latest", "name", "prerelease", "prerelease-identifier", "publish", "tag", "token", "version"

(action)